x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -332,8 +332,8 @@ const getAuthenticationColumns = (startDate: number): Array<Columns<Authenticati
     truncateText: false,
     hideForMobile: false,
     render: ({ node }) => {
-      const hostName: string | null = get('lastSuccess.host.name', node);
-      const hostId: string | null = get('lastSuccess.host.id', node);
+      const hostName: string | null | undefined = get('lastSuccess.host.name', node);
+      const hostId: string | null | undefined = get('lastSuccess.host.id', node);
```

Comment:
In the `authentications_table/mock.ts` host is mocked as:

``` typescript
host: {
  id: ['host-id-1'],
  name: ['host-1'],
}
```

Should `host.name`/`host.id` have a zero index references as well?

Reply:
I forgot `lastFailure.host.name` as well so I changed these as well as `lastFailure`